RELEASE/scripts/autoscend/combat/auto_combat_util.ash

```diff
@@ -174,6 +195,44 @@ string useItems(item it1, item it2)
 	return useItems(it1, it2, true);
 }
 
+skill getSniffer(monster enemy, boolean inCombat)
+{
+	//returns the skill we want to use to sniff the enemy
+	//all sniffers eliminate the rule that reduces the odds of encountering the same enemy twice in a row.
```

Comment:
Believe olfaction is the only sniffer that eliminates the combat queue rejection rule

Reply:
Long Con does too.

Reply:
thank you. fixed these comments